users/service.go

```diff
@@ -106,33 +107,8 @@ type Service interface {
 	//SendPasswordReset sends reset password link to email.
 	SendPasswordReset(ctx context.Context, host, email, token string) error
 
-	// CreateGroup creates new user group.
-	CreateGroup(ctx context.Context, token string, group Group) (Group, error)
-
-	// UpdateGroup updates the group identified by the provided ID.
-	UpdateGroup(ctx context.Context, token string, group Group) error
-
-	// ViewGroup retrieves data about the group identified by ID.
-	ViewGroup(ctx context.Context, token, id string) (Group, error)
-
-	// ListGroups retrieves groups that are children to group identified by parenID
-	// if parentID is empty all groups are listed.
-	ListGroups(ctx context.Context, token, parentID string, offset, limit uint64, m Metadata) (GroupPage, error)
-
-	// Members retrieves users that are assigned to a group identified by groupID.
-	ListMembers(ctx context.Context, token, groupID string, offset, limit uint64, m Metadata) (UserPage, error)
-
-	// ListMemberships retrieves groups that user identified with userID belongs to.
-	ListMemberships(ctx context.Context, token, groupID string, offset, limit uint64, m Metadata) (GroupPage, error)
-
-	// RemoveGroup removes the group identified with the provided ID.
-	RemoveGroup(ctx context.Context, token, id string) error
-
-	// Assign adds user with userID into the group identified by groupID.
-	Assign(ctx context.Context, token, userID, groupID string) error
-
-	// Unassign removes user with userID from group identified by groupID.
-	Unassign(ctx context.Context, token, userID, groupID string) error
+	// ListMembers retrieves everything that is assigned to a group identified by groupID.
+	ListMembers(ctx context.Context, token string, group groups.Group, offset, limit uint64, m Metadata) (UserPage, error)
```

Comment:
If all other vars are called with more than one letter, prefer `meta` over `m`.

Reply:
Plus that way you will stay consistent to what you used above.

Reply:
Consider passing a single structure `PageMetadata` that contains `offset`, `limit` and `metadata`